scripts/ivy/build.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+set -e
+
+outputPath=""
+updateAngular=""
+skipCdk=0
+
+for i in "$@"
+do
+  case $i in
+    --update-angular=*)
+    eval updateAngular="${i#*=}"
+    shift
+    ;;
+
+    --skip-cdk)
+    skipCdk=1
+    shift
+    ;;
+
+    *)
+    ;;
+  esac
+done
+
+function prep() {
+  echo ">>> Preparing node_modules"
+  chmod -R u+w node_modules/@angular
+  rm -rf node_modules/@angular/*
+}
+
+function buildNgPackages() {
+  ngDir=$1
+  echo ">>> Building @angular packages (from $ngDir)"
+  pushd $ngDir
+  bazel build --define=compile=local //packages/{animations,core,common,compiler,compiler-cli,forms,platform-browser}:npm_package
+  outputPath=`bazel info 2>&1 | grep output_path | cut -d ':' -f 2 | cut -c 2-`/darwin-fastbuild/bin/packages
+  popd
+}
+
+function installNgPackage() {
+  name=$1
+  echo "    @angular/$name"
+  cp -r $outputPath/$name/npm_package node_modules/@angular/$name
+}
+
+function buildPackage() {
+  name=$1
+  echo "    $name"
+  
+  # First, fix the build definition.
+  # Update the tsconfig to compile from index.ts instead of public-api.ts, and turn on Ivy options.
+  node scripts/ivy/fix-tsconfig.js src/$name/tsconfig-build.json
+
+  # If no index.ts exists, rename public-api.ts.
+  if [ ! -f src/$name/index.ts ]
+  then
+    mv src/$name/public-api.ts src/$name/index.ts
+  fi
+
+  node_modules/.bin/ngc -p src/$name/tsconfig-build.json
+}
+
+if [ "$updateAngular" != "" ]
+then
+  prep
```

Comment:
Not that it matters too much, but I think its possible that someone could: 
```
scripts/ivy/build.sh --update-angular=/path/to/angular --skip-cdk
```
which would remove everything from `@angular/*` and then fail to create the CDK files which material relies on.

Though, this is also a superuser type script where the user should know whats going on, so I don't know that it matters too much.